scratch3/blocks.js

```diff
@@ -801,7 +801,11 @@ class DocumentView {
     var svg = SVG.newSVG(width, height, this.scale)
     svg.appendChild((this.defs = SVG.withChildren(SVG.el("defs"), makeIcons())))
 
-    svg.appendChild(SVG.group(elements))
+    svg.appendChild(
+      SVG.group(elements, {
+        style: `transform: scale(${this.scale})`,
+      }),
```

Comment:
nit: I think the intent is you write
```suggestion
      SVG.setProps(SVG.group(elements), {
        style: `transform: scale(${this.scale})`,
      })),
```
Your proposed change to `SVG.group` may well be more ergonomic, but in that case I'd want to review each caller of it and update them appropriately